entity-service-impl/src/test/java/org/hypertrace/entity/query/service/EntityQueryServiceImplTest.java

```diff
@@ -196,10 +201,149 @@ public void testUpdate_success() throws Exception {
             });
 
     verify(mockEntitiesCollection, times(1))
-        .updateSubDoc(
-            eq(new SingleValueKey("tenant1", "entity-id-1")),
-            eq("attributes.status"),
-            eq(new JSONDocument(DocStoreJsonFormat.printer().print(newStatus))));
+        .bulkUpdateSubDocs(
+            eq(
+                Map.of(
+                    new SingleValueKey("tenant1", "entity-id-1"),
+                    Map.of(
+                        "attributes.status",
+                        new JSONDocument(DocStoreJsonFormat.printer().print(newStatus))))));
+  }
+
+  @Nested
+  class testBulkUpdate {
```

Comment:
Class names are camel cased, starts with an upper case letter

```suggestion
  class BulkUpdateEntities {
```